frontends/verific/verific.cc

```diff
@@ -2470,6 +2470,9 @@ struct VerificPass : public Pass {
 					worker.run(nl);
 			}
 
+			for (auto nl : nl_todo)
+				nl->ChangePortBusStructures(1 /* hierarchical */);
```

Comment:
We probably want to add something similar around line 1897.

Reply:
Roger that. Done in 7b445121c. `./yosys -Qp "read -sv bug69.sv; hierarchy -top mcve; dump i:in*"` now behaves as `verific -sv bug69.sv; verific -import -all` does